src/analysis/processing/qgsalgorithmwritevectortiles.cpp

```diff
@@ -157,7 +157,12 @@ void QgsWriteVectorTilesMbtilesAlgorithm::initAlgorithm( const QVariantMap & )
   addParameter( new QgsProcessingParameterString( QStringLiteral( "META_DESCRIPTION" ), QObject::tr( "Metadata: Description" ), QVariant(), false, true ) );
   addParameter( new QgsProcessingParameterString( QStringLiteral( "META_ATTRIBUTION" ), QObject::tr( "Metadata: Attribution" ), QVariant(), false, true ) );
   addParameter( new QgsProcessingParameterString( QStringLiteral( "META_VERSION" ), QObject::tr( "Metadata: Version" ), QVariant(), false, true ) );
-  addParameter( new QgsProcessingParameterString( QStringLiteral( "META_TYPE" ), QObject::tr( "Metadata: Type" ), QVariant(), false, true ) );
+  addParameter( new QgsProcessingParameterEnum( QStringLiteral( "METADATA_TYPE" ), QObject::tr( "Metadata: Type" ), QStringList() << QStringLiteral( "overlay" ) << QStringLiteral( "baselayer" ), false, QVariant(), true ) );
```

Comment:
instead of a new parameter you could leave the existing string parameter and set the widget metadata "value_hints" to show a combobox instead.

Reply:
I was thinking about it, but with this approach it is possible to pass invalid value, e.g. when running from Python code so I choose new parameter instead. If you think it is better to have existing parameter with value hints added, I will update PR.

Reply:
> If you think it is better to have existing parameter with value hints added

I'd probably still lean that way. It still guides users to choosing valid values, while allowing scripts to have a bit more flexibility if they have some reason to deviate from the specs

Reply:
Ok, I have changed it to use `value_hints`.